agents/sequential.py

```diff
@@ -0,0 +1,77 @@
+from agents.base import BaseAgent
+from selenium.webdriver.common.keys import Keys 
+
+
+class SequentialAgent(BaseAgent):
+    """
+    Class object for the sequential agent.
+
+    This agent is governed by simple logic. Given a starting maneuver (down,
+    left, up, right) and a direction (clockwise/counterclockwise) the agent
+    sequentially makes those maneuvers.
+
+    Parameters
+    ----------
+    initial_move:   unicode literal, default 0 (move_up)
```

Comment:
Default value is now `Keys.ARROW_UP` - one of the direct Selenium WebDriver common keys instead of having encoded maneuvers which need to later be translated into the keys.